modules/virt-installing-virtctl-client.adoc

```diff
@@ -1,66 +1,75 @@
 // Module included in the following assemblies:
 //
-// virt/install/virt-enabling-virtctl.adoc
+// * virt/install/virt-installing-virtctl.adoc
 
 :_content-type: PROCEDURE
 [id="virt-installing-virtctl-client_{context}"]
-= Installing the virtctl client
+= Installing the virtctl client on Linux, Windows, and macOS
 
-Extract and install the `virtctl` client after downloading from the appropriate location for your operating system.
+Download and install the `virtctl` client for your operating system.
 
-.Prerequisites
+.Procedure
 
-* You must have downloaded the `virtctl` client.
+. Navigate to *Virtualization > Overview* in the {product-title} web console.
 
-.Procedure
+. Click the *Download virtctl* link on the upper right corner of the page and download the `virtctl` client for your operating system.
+
+. Install `virtctl`:
 
 * For Linux:
 
-. Extract the tarball. The following CLI command extracts it into the same directory as the tarball:
+.. Decompress the archive file:
 +
 [source,terminal]
 ----
 $ tar -xvf <virtctl-version-distribution.arch>.tar.gz
 ----
 
-. Navigate the extracted folder hierachy and run the following command to make the `virtctl` binary executable:
+.. Run the following command to make the `virtctl` binary executable:
 +
 [source,terminal]
 ----
-$ chmod +x <virtctl-file-name>
+$ chmod +x <path/virtctl-file-name>
 ----
 
-. Move the `virtctl` binary to a directory in your `PATH` environment variable.
-
-. To check your path, run the following command:
+.. Move the `virtctl` binary to a directory in your `PATH` environment variable.
++
+You can check your path by running the following command:
 +
 [source,terminal]
 ----
 $ echo $PATH
 ----
 
-* For Windows users:
-
-. Unpack and unzip the archive.
+.. Set the `KUBECONFIG` environment variable:
```

Comment:
@dominikholler Please review for [GH#53361](https://github.com/openshift/openshift-docs/issues/53361) .